src/examples/Makefile

```diff
@@ -0,0 +1,57 @@
+# Build examples using make
+# Target OS: Linux
+# Compiler: gcc
+# Target processor: x86_64
+
+# Paths to relevant programs
+CXX            = g++
+LINKER         = $(CXX)
+
+# Compiler Flags
+ABI_FLAGS      = -fstack-protector -m64 -pthread
+LANG_FLAGS     = -std=c++17 -D_REENTRANT
+LANG_EXE_FLAGS =
+CXXFLAGS       = -O3
+WARN_FLAGS     = -Wall -Wextra -Wpedantic -Wstrict-aliasing -Wcast-align -Wmissing-declarations -Wpointer-arith -Wcast-qual -Wzero-as-null-pointer-constant -Wnon-virtual-dtor -Wold-style-cast -Wsuggest-override -Wshadow -Wextra-semi -Werror -Wno-error=strict-overflow -Wno-error=zero-as-null-pointer-constant -Wno-error=non-virtual-dtor
```

Comment:
For the sake of readability: Please add `-Wno-unused-parameter` and remove the commenting of unused functions parameters in the examples.

Reply:
changed